tests/test_worker.py

```diff
@@ -105,9 +105,9 @@ def test_worker_stop_all(hq_env: HqEnv):
     hq_env.start_server()
     processes = [hq_env.start_worker() for _ in range(4)]
 
-    wait_for_worker_state(hq_env, [1, 2, 3, 4], ["RUNNING" for _ in range(4)]),
+    (wait_for_worker_state(hq_env, [1, 2, 3, 4], ["RUNNING" for _ in range(4)]),)
     hq_env.command(["worker", "stop", "all"])
-    wait_for_worker_state(hq_env, [1, 2, 3, 4], ["STOPPED" for _ in range(4)]),
+    (wait_for_worker_state(hq_env, [1, 2, 3, 4], ["STOPPED" for _ in range(4)]),)
```

Comment:
Extra comma after the line, now it is more visible and we should remove it.

Reply:
Thanks, fixed.